libsrc/atari/targetutil/w2cas.c

```diff
@@ -35,7 +35,7 @@ static struct __iocb *findfreeiocb(void)
 static void exitfn(void)
 {
     /* if DOS will automatically clear the screen, after the program exits, wait for a keypress... */
-    if (! _is_cmdline_dos())
+    if (doesclrscrafterexit())
         cgetc();
 }
```

Comment:
Given that the return value of doesclrscrafterexit() doesn't change during runtime of the program it may be even "cooler" to just do

```
if (doesclrscrafterexit())
    atexit(cgetc);
```